### DISCUSSION TITLE
How to Transition from listening to inactive and back to listening through method call

### DISCUSSION THREAD

**Discussion by kaza** [KoljaB/RealtimeSTT#55] [Q&A] [answered]
Hi, 

I would like to be able to switch from listening to inactive mode on demmand. This is actually same as saying a "wake_word" and what happens afterwards. 

I tried using abort and shutdown but that did not work. any suggestions would be highly appreciated 

of course I could initiate recorder and distroy it, but that takes 8 seconds on my machine so I would like to be able to switch between the states faster.

thanks!

**Comment by KoljaB** [comment]
Hey, please set `recorder.wake_word_activation_delay = 0` to disable and set it back to your default value to enable wake words at runtime.

**Comment by kaza** [comment, marked as answer]
Awsome, I tested it and it wors, thank you!

here is a code snipet which I used for testing`
```
    with AudioToTextRecorder(spinner=False, model="medium", language="en",
                             level=logging.INFO) as recorder:
        while True:
            print("### the below line will put the recorder into listening state ")
            recorder.wake_word_activation_delay = 0
            result = recorder.text()
            print("Result of listening:" + result)
            print("### moving to inactive state  now")
            recorder.wake_word_activation_delay = 100
            time.sleep(10) # not listening during playback for example
            print("### next loop ### ")
````
let me know if you want me to create a tiny example as PR
